usaspending_api/references/management/commands/load_cgacs.py

```diff
@@ -3,7 +3,6 @@
 
```

Comment:
Isn't the model being removed in this PR? Can we drop this loader?

Reply:
I wasn't planning on putting the new loader into regular use yet since we're not really using any of the features yet, but I suppose we can.

Also, model got moved not removed.  I need to clean up how I'm doing that.  I don't like the  current recreate method.

Reply:
I think I'm going to stick with my original thought and hold off on putting the new loader into production yet.  Everything's in a half state where we have two sets of toptier/subtier agencies.  It's just going to get confusing and create extra work updating the nightly pipeline multiple times.  We can remove the `load_cgacs` as part of the followup ticket where we start folding this new agency stuff into the rest of the app.